internal/services/apimanagement/api_management_logger_resource.go

```diff
@@ -281,6 +324,12 @@ func flattenApiManagementLoggerEventHub(d *pluginsdk.ResourceData, properties *l
 			if conn, ok := existingEventHub["connection_string"]; ok {
 				eventHub["connection_string"] = conn.(string)
 			}
+			if endpoint, ok := existingEventHub["endpoint_uri"]; ok {
+				eventHub["endpoint_uri"] = endpoint
+			}
+			if clientId, ok := existingEventHub["client_id"]; ok {
```

Comment:
Then we only set to state if it isn't `SystemAssigned`
```suggestion
			if clientId, ok := existingEventHub["client_id"]; ok {
				if clientId != "SystemAssigned" {
					eventHub["client_id"] = clientId
				}
```